### ISSUE TITLE
Show more debonding information

### ISSUE THREAD

**Issue by matevz** [oasisprotocol/oasis-wallet-web#420]
There is very little information about the debonding period when you want to reclaim your stake. I suggest we provide the user with basic debonding information:
- [ ] model: fetch current epoch, find a way to compute expected epoch duration (genesis file?)
- [ ] in the `reclaimEscrow` preview dialog, there should be a note saying `Current debonding period is x epochs (~y days z hours). You will not be able to access the reclaimed balance until the debonding period is over. Reclaimed balance is not awarded during the debonding period.`
- [ ] in Debonding delegations table add a localized `(~dd-mm-yy)` to values in `Debonding epoch` columns
- [ ] in Debonding delegations table add `Remaining epochs` column showing `x epochs (~y days z hours)`

**Comment by tjanez** [comment]
> model: fetch current epoch, find a way to compute expected epoch duration (genesis file?)

For the expected epoch duration, I think one would need to query Oasis Monitor for the average block length and compute that by multiplying with the expected number of blocks in an epoch.

**Comment by tjanez** [comment]
> Current debonding period is x epochs (~y days z hours). You will not be able to access the reclaimed balance until the debonding period is over. Reclaimed balance is not awarded during the debonding period.

Is there a gRPC query to get the current value of the `staking.params.debonding_interval` parameter?

**Comment by pro-wh** [comment]
yeah, use StakingConsensusParameters https://github.com/oasisprotocol/oasis-sdk/blob/7d7d592b533cc4c43cde39d9c8cc5440958491d8/client-sdk/ts-web/core/src/client.ts#L990
